plugins/reporters/static-html/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -183,17 +183,17 @@
     border-right: 1px solid rgba(34, 36, 38, .15);
 }
 
-.ort-report-table.ort-report-rule-violation-table tr.ort-resolved {
+.ort-report-rule-violation-table tr.ort-resolved {
```

Comment:
Commit message: In the first sentence, the "specializations" / "spezialize" duplication reads a bit odd, IMO. Can you think of a way to reword? Maybe just say "refers to" instead of "spezialize"? In any case, it should say "specialize".

Reply:
I ended up re-wording it to avoid the term "specialize" entirely.